packages/theme-chalk/src/upload.scss

```diff
@@ -84,7 +88,8 @@
 }
 
 @include b(upload-dragger) {
-  padding: 40px 10px;
+  padding: var(--el-upload-dragger-padding-horizontal)
```

Comment:
Please use `getCssVar` to generate css var. (So we can change namespace).

Just refer https://github.com/element-plus/element-plus/pull/7344/